### DISCUSSION TITLE
How to host next js app in sub folder under existing asp.net web application running on iis

### DISCUSSION THREAD

**Discussion by itsmemittal** [vercel/next.js#15930] [Help]
Hi i have develop my next JS application i need to host it under sub folder of my existing web application under IIS.

OS : Windows
WebServer : IIS
Exsting web application : ASP.NET hosted on IIS

**Comment by itsmemittal** [comment]
current app is not static app its has dynamic data calling from api and rendering in next js app.

**Comment by anhhtca** [comment]
Hello,
Have you done with that implementation ?
